apogee/tools/__init__.py

```diff
@@ -174,3 +188,144 @@ def toApStarGrid(spec):
         return out[0]
     else:
         return out
+
+totalpix = 8575 # total pixels on the apstar grid
```

Comment:
Can we replace all of this with a line similar to:

https://github.com/jobovy/apogee/blob/master/apogee/spec/plot.py#L85

Better would be to move that apStarWavegrid function to this file (where it more naturally belongs); we could import it into apogee.spec.plot for backwards compatibility.